src/BSeries.jl

```diff
@@ -612,6 +615,74 @@ function bseries(ros::RosenbrockMethod, order)
     return series
 end
 
+"""
+    ContinuousStageRungeKuttaMethod
+
+A struct that describes a CSRK method. This kind of 'struct' should be constructed 
+via [`CSRK`] 
+        'csrk = CSRK(M)' 
+in order to later call the 'bseries' function. 
+"""
```

Comment:
It would be nice to include at least a reference to the paper of Miyatake and Butcher here, too.